Libraries/StyleSheet/processFontVariant.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import type {____FontVariantArray_Internal} from './StyleSheetTypes';
+
+function processFontVariant(
+  fontVariant: ____FontVariantArray_Internal | string,
+): ?____FontVariantArray_Internal {
+  if (Array.isArray(fontVariant)) {
+    return fontVariant;
+  }
+
+  // $FlowFixMe[incompatible-type]
+  const match: ?____FontVariantArray_Internal = fontVariant.match(
+    new RegExp(/([a-zA-Z'-]+)/gm),
```

Comment:
Quoting @javache:

> Can we avoid regular expressions here since it's just a string split by space? This also seems like it would cause the regex to be re-compiled on every process call.

Reply:
Sure, let me update this

Reply:
FIxed @cipolleschi